templates/index.html

```diff
@@ -5,45 +5,71 @@ <h2 class="subtitle">A post-modern <strong>text editor</strong>.</h2>
       <!-- <img src="./screenshot.png" /> -->
       <asciinema-player src="/430253.cast" cols="120" rows="35"></asciinema-player>
       <div class="action">
-        <a href="https://github.com/helix-editor/helix">Install now.</a>
+        <a href="https://docs.helix-editor.com/install.html">Install now.</a>
       </div>
       <hr/>
       <section class="content">
-        <h2><strong>Multiple selections</strong> by default, based on <a href="http://kakoune.org/">Kakoune</a>.</h2>
-          <p>The whole design is based around multiple selections as an editing
-          primitive, with a single cursor just being a subset of that. Commands
-          manipulate selections which then allow you to concurrently edit code.</p>
-          <p>Editing is focused on being easy to reason with, even if it takes
-          a keystroke or two more than Vim or Kakoune.</p>
-          <h2><strong>Syntax highlighting</strong> and <strong>code editing</strong> using <a href="https://tree-sitter.github.io/tree-sitter/">Tree-sitter</a>.</h2>
-          <p>No more slow regex highlighting! Tree-sitter parses code into
-          syntax trees much like a compiler, giving us a lot more information
-          about code structure. We can track local variables, calculate
-          indentation and manipulate selection to select syntax nodes.</p>
-          <p>Parsing is incremental and fast enough to run on every keystroke.</p>
-          <p>Robust enough to produce results even with syntax errors.</p>
-          <h2>Built-in <strong>language server</strong> support.</h2>
-          <ul>
-            <li>Context aware completion</li>
-            <li>Documentation</li>
-            <li>Diagnostics</li>
-            <li>Code actions</li>
-          </ul>
-          <h2>Runs in a <strong>terminal</strong>.</h2>
-          <p>The base implementation uses a terminal frontend and combines well with tmux or usage over ssh.</p>
-          <p>I'm also exploring alternate frontends (wgpu-rs, skia/skulpin based
-          rendering).</p>
-
-          <h2>Built with <strong>Rust</strong>. No Electron. No VimScript. No JavaScript.</h2>
-          <p>Your laptop battery life will thank you.</p>
+        <div class="selling-points">
+          <div class="selling-point">
+            <h2>Multiple selections</h2>
+            Multiple cursors as a core editing primitive, inspired from
```

Comment:
```suggestion
            Multiple cursors as a core editing primitive, inspired by
```

I think this reads better but I would be ok with `from` as well